velox/docs/functions/spark/string.rst

```diff
@@ -427,3 +427,36 @@ String Functions
     Returns string with all characters changed to uppercase. ::
 
         SELECT upper('SparkSql'); -- SPARKSQL
+
+.. spark:function:: varchar_type_write_side_check(string, limit) -> varchar
+
+    An implementation for Spark's varcharTypeWriteSideCheck function of class CharVarcharCodegenUtils.
+    Removes trailing 0x20(space) characters from ``string`` to fit in the ``limit``, if not fit, throws exception. ::
```

Comment:
Can you provide some example?

Reply:
I see. Here we provide SQL examples to illustrate how to use the function in Spark, while this function cannot be called directly in SQL. Please document that information.